tests/testthat/test-verb-expand.R

```diff
@@ -0,0 +1,138 @@
+test_that("expand completes all values", {
+  skip_if_not_installed("tidyr")
```

Comment:
In tests, you can just assume that suggested packages are installed, so you can remove all these skips. I'd also have a minor preference to avoid `local_package()`, just because it only saves a small amount of typing.

Reply:
I am not sure whether I understood your point about `local_package()` completely. I replaced it with
`suppressPackageStartupMessages(library(tidyr))` and `detach("package:tidyr")`. Is this what you had in mind?

Reply:
No, I mean just don't use it and use `::` instead

Reply:
oh, oh, well, that would have been too obvious... Sorry about that!